src/main/java/tc/oc/pgm/commands/RotationCommands.java

```diff
@@ -0,0 +1,247 @@
+                ? ChatColor.DARK_AQUA.toString() + index
+                : String.valueOf(index);
+        return (indexString) + ". " + ChatColor.RESET + map.getInfo().getShortDescription(sender);
+      }
+    }.display(audience, maps, page);
+  }
+
+  @Command(
+      aliases = {
+        "rotationcheckout",
+        "rotationcheck",
+        "rotcheckout",
+        "rotcheck",
+        "rtcheckout",
+        "rtcheck"
+      },
+      desc = "Shows the maps in the specified rotation",
+      usage = "[rotation] [page]",
+      help = "Shows all the maps that are currently in a specific rotation.")
+  public static void rotationCheckout(
+      Audience audience,
+      CommandSender sender,
+      MatchManager matchManager,
+      String rotationName,
+      @Default("1") int page)
+      throws CommandException {
+
+    List<PGMMap> maps;
+    Rotation rotation = matchManager.getRotationManager().getRotationByName(rotationName);
+    if (rotation != null) maps = rotation.getMaps();
+    else {
+      sender.sendMessage(
+          ChatColor.RED
+              + AllTranslations.get().translate("command.rotation.noRotationMatch", sender));
+      return;
+    }
+
+    int resultsPerPage = 8;
+    int pages = (maps.size() + resultsPerPage - 1) / resultsPerPage;
+
+    String listHeader =
+        ChatColor.BLUE.toString()
+            + ChatColor.STRIKETHROUGH
+            + "-----------"
+            + ChatColor.RESET
+            + " "
+            + AllTranslations.get().translate("command.rotation.currentRotation.title", sender)
+            + ChatColor.DARK_AQUA
+            + " ("
+            + ChatColor.AQUA
+            + rotation.getName()
+            + ChatColor.DARK_AQUA
+            + ")"
+            + " ("
+            + ChatColor.AQUA
+            + page
+            + ChatColor.DARK_AQUA
+            + " of "
+            + ChatColor.AQUA
+            + pages
+            + ChatColor.DARK_AQUA
+            + ") "
+            /*
+             * This below ensures that the string remains always colored. For some reason the logic behind this de-colored
+             * the text if it were to get put into a second line of chat when the header exceeded the maximum length
+             * of the first one
+             */
+            + ChatColor.translateAlternateColorCodes('&', "&9&m-----------");
+
+    new PrettyPaginatedResult<PGMMap>(listHeader, resultsPerPage) {
+      @Override
+      public String format(PGMMap map, int index) {
+        index++;
+        String indexString =
+            rotation.getPosition() + 1 == index
+                ? ChatColor.DARK_AQUA.toString() + index
+                : String.valueOf(index);
+        return (indexString) + ". " + ChatColor.RESET + map.getInfo().getShortDescription(sender);
+      }
+    }.display(audience, maps, page);
+  }
+
+  @Command(
+      aliases = {"rotations", "rots", "rts"},
```

Comment:
I'd say support `rotation` and `rot` command aliases, not `rt`. (this is for all of the commands)

Reply:
These ones should be fine. https://github.com/Electroid/PGM/pull/114/commits/c569a6d24c1e086bf9568de2e589ac3f3930161d